CSS/style.css

```diff
@@ -0,0 +1,75 @@
+body{
+    background-image: url(https://slack-imgs.com/?c=1&o1=ro&url=https%3A%2F%2Fimg.freepik.com%2Ffree-photo%2Fdelicious-ice-cream-cones_23-2149460312.jpg%3Fw%3D2000);
+    background-repeat: no-repeat;
+    background-size:cover;
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+    color: #292F36;
+    margin: 20px 50px;
+    text-align: initial;
+    height:auto;
+    
+}
+
+
+#site-title{
+    font-size: 80px;
+    color: #6F73D2;
+}
+
+
+header,h1{
+    color:#83C9F4;
+}
+
+.container {
+    display: flex;
+    margin: 50px;
+
+}
+
+.container p {
+    margin: 10px 50px;
+    padding: 20px;
+    max-width: 500px;
+}
+
+ul{
+    list-style: none;
+}
+a:link{
+    color:white;
+}
+.btn {
+  background: #3498db;
+  background-image: -webkit-linear-gradient(top, #3498db, #2980b9);
+  background-image: -moz-linear-gradient(top, #3498db, #2980b9);
+  background-image: -ms-linear-gradient(top, #3498db, #2980b9);
+  background-image: -o-linear-gradient(top, #3498db, #2980b9);
+  background-image: linear-gradient(to bottom, #3498db, #2980b9);
+  -webkit-border-radius: 28;
+  -moz-border-radius: 28;
+  border-radius: 28px;
+  font-family: Arial;
+  color: #ffffff;
+  font-size: 20px;
+  padding: 10px 20px 10px 20px;
+  text-decoration:none;
+  margin: 10px;
+  outline:none;
+  border:none;
+}
+
+.btn:hover {
+  background: #3cb0fd;
+  background-image: -webkit-linear-gradient(top, #3cb0fd, #3498db);
+  background-image: -moz-linear-gradient(top, #3cb0fd, #3498db);
+  background-image: -ms-linear-gradient(top, #3cb0fd, #3498db);
+  background-image: -o-linear-gradient(top, #3cb0fd, #3498db);
+  background-image: linear-gradient(to bottom, #3cb0fd, #3498db);
```

Comment:
Only the last background-image property is being used! All of the others can be deleted.